src/Components/CollectionsRail/CollectionsRail.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Col, Grid, Row, Sans, Spacer } from "@artsy/palette"
+import { CollectionsRail_collections } from "__generated__/CollectionsRail_collections.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+
+import { CollectionEntity } from "./Collection"
+
+interface CollectionRailsProps {
+  collections?: CollectionsRail_collections
+  // collections?: any
+}
+
+const RailsWrapper = styled(Box)`
+  width: 100%;
+  max-width: 1250px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+`
+
+const CollectionsRow = styled(Row)`
+  justify-content: space-between;
+`
+
+export class CollectionsRail extends React.Component<CollectionRailsProps> {
+  getRails = () => {
+    const collections: any[] = this.props.collections.map(
+      (collection, index) => {
+        return (
+          <Col xl={5} lg={5} md={5} sm={12} key={index}>
```

Comment:
Typically grid props should be reversed, from smallest to largest,  and it will take the lowest match and apply to all larger breakpoints. So, `md={5}` automatically applies 5 to lg and xl. Going in the opposite direction, down to sm and xs, the default is 12 columns. So this can be simplified to: 

```
<Col md={5}>
   ...
</Col>
```

(sorry, this is all kinda confusing! I can explain more if need be)